mos-platform/c64/ldscripts/kernal.ld

```diff
@@ -0,0 +1,39 @@
+PROVIDE(CINT   = 0XFF81);
```

Comment:
Make sure to use reserved symbols for these; otherwise, we run afoul of C standards compliance. The standard library might incorrectly call user code (or worse, call a user _variable_) if they define one of these symbols.